packages/pn-commons/src/utility/index.ts

```diff
@@ -1,6 +1,8 @@
 import { Configuration } from '../services/configuration.service';
+import EventStrategyFactory from '../utility/MixpanelUtils/EventStrategyFactory';
 import { AppError, AppErrorFactory, UnknownAppError, errorFactoryManager } from './AppError';
 import { AppResponsePublisher, ResponseEventDispatcher } from './AppResponse';
+import EventStrategy from './MixpanelUtils/EventStrategy';
```

Comment:
If we move the EventStrategy in the models dir, we have to move this import and below export too